src/library_syscall.js

```diff
@@ -660,7 +666,9 @@ var SyscallsLibrary = {
   __syscall140: function(which, varargs) { // llseek
     var stream = SYSCALLS.getStreamFromFD(), offset_high = SYSCALLS.get(), offset_low = SYSCALLS.get(), result = SYSCALLS.get(), whence = SYSCALLS.get();
     var offset = offset_low;
+#if WASM_BACKEND == 0
```

Comment:
I think for asm.js and wasm32 in general we don't need the higher bits anyhow, so how about making the assert unconditional? less differences between the wasm backend and others is better.